.github/workflows/push.yml

```diff
@@ -28,9 +26,12 @@ jobs:
 
     - name: Run tests
       run: make test
-
+    - name: Get python version
+      run: |
+        python_version=$(python  -V | cut -d' ' -f2)
+        echo "python_version=${python_version}" >> $GITHUB_ENV
     - name: Copy site-packages in workspace
-      working-directory: /opt/hostedtoolcache/Python/3.9.9/x64/lib/python3.9/
+      working-directory: /opt/hostedtoolcache/Python/${{ env.python_version }}/x64/lib/python3.9/
```

Comment:
This is a good idea!

Reply:
It failed possible due to change in python version.. see https://github.com/cds-snc/notification-document-download-api/runs/5205000366?check_suite_focus=true